test/e2e-common.sh

```diff
@@ -333,7 +333,7 @@ function install_channel_crds() {
   if [[ -z "${EVENTING_IN_MEMORY_CHANNEL_YAML:-}" ]]; then
     build_knative_from_source
   else
-    echo "use exist EVENTING_SUGAR_CONTROLLER_YAML"
+    echo "use exist EVENTING_IN_MEMORY_CHANNEL_YAML"
```

Comment:
Nit: would be a bit more useful to print the actual value of the variable?

```suggestion
    echo "use existing ${EVENTING_IN_MEMORY_CHANNEL_YAML}"
```

/lgtm
/hold